widgetastic_manageiq/__init__.py

```diff
@@ -5100,9 +5102,78 @@ class MigrationDashboardStatusCard(AggregateStatusCard):
     ROOT = ParametrizedLocator(
         './/div[contains(@class, "card-pf-aggregate-status")'
         ' and not(contains(@class, "card-pf-aggregate-status-mini")) and'
-        ' h2[contains(@class,"card-pf-title")  and contains(.,{@name|quote})]]'
+        ' h2[contains(@class,"card-pf-title")  and contains(normalize-space(.),{@name|quote})]]'
     )
-    TITLE_ANCHOR = ".//h2"
+    TITLE_ANCHOR = "./h2"
+
+    @property
+    def is_active(self):
+        return "active" in self.root_browser.classes(self.ROOT or self.__locator__)
+
+
+class V2VPaginator(Paginator):
+    """ Represents Paginator control for V2V."""
+
+    PAGINATOR_CTL = './/div[contains(@class,"form-group")][./ul]'
+    './input[contains(@class,"pagination-pf-page")]'
```

Comment:
Please update the formatting/syntax here, this hanging un-indented string is not clear. Wrap with `()`, or start it on a newline so it all fits in one line